src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $color-bg-white;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  box-sizing: border-box;
+  position: relative;
+  padding: 16px;
+
+  &__image {
+    display: block;
+    background-image: url(../images/imac.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 20px;
```

Comment:
The use of arbitrary margin values (16px 3px 20px) can lead to inconsistency across the design. Ensure that the margins align with the design requirements and try to use variables for consistent spacing throughout the styles.